webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -461,14 +458,13 @@ def mocked_supersearch_get(**params):
         assert res["facets"]
 
         # Verify forbidden fields are not exposed.
-        assert "exploitability" not in res["hits"]
         assert "url" not in res["hits"]
 
         # Verify it's not possible to use restricted parameters.
         response = self.client.get(
             url,
             {
-                "exploitability": "high",
+                "url": "example.com",
```

Comment:
There were a bunch of permissions tests. I swapped out the `exploitability` which required `view_exploitability` for `url` and `user_comments` and `gmp_plugin_library` which require `view_pii`.